contributors/design-proposals/pod-security-policy.md

```diff
@@ -0,0 +1,351 @@
+PodSecurityPolicy will reference users and groups that are allowed
+to operate under the constraints.  In order to support this, `ServiceAccounts` must be mapped
+to a user name or group list by the authentication/authorization layers.  This allows the security
+context to treat users, groups, and service accounts uniformly.
+
+Below is a list of PodSecurityPolicies which will likely serve most use cases:
+
+1.  A default policy object.  This object is permissioned to something which covers all actors, such
+as a `system:authenticated` group, and will likely be the most restrictive set of constraints.
+1.  A default constraints object for service accounts.  This object can be identified as serving
+a group identified by `system:service-accounts`, which can be imposed by the service account authenticator / token generator.
+1.  Cluster admin constraints identified by `system:cluster-admins` group - a set of constraints with elevated privileges that can be used
+by an administrative user or group.
+1.  Infrastructure components constraints which can be identified either by a specific service
+account or by a group containing all service accounts.
+
+```go
+// PodSecurityPolicy governs the ability to make requests that affect the SecurityContext
+// that will be applied to a pod and container.
+type PodSecurityPolicy struct {
+	unversioned.TypeMeta `json:",inline"`
+	api.ObjectMeta       `json:"metadata,omitempty"`
+
+	// Spec defines the policy enforced.
+	Spec PodSecurityPolicySpec `json:"spec,omitempty"`
+}
+
+// PodSecurityPolicySpec defines the policy enforced.
+type PodSecurityPolicySpec struct {
+	// Privileged determines if a pod can request to be run as privileged.
+	Privileged bool `json:"privileged,omitempty"`
+	// Capabilities is a list of capabilities that can be added.
+	Capabilities []api.Capability `json:"capabilities,omitempty"`
+	// Volumes allows and disallows the use of different types of volume plugins.
+	Volumes VolumeSecurityPolicy `json:"volumes,omitempty"`
+	// HostNetwork determines if the policy allows the use of HostNetwork in the pod spec.
+	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// HostPorts determines which host port ranges are allowed to be exposed.
+	HostPorts []HostPortRange `json:"hostPorts,omitempty"`
+	// HostPID determines if the policy allows the use of HostPID in the pod spec.
+	HostPID bool `json:"hostPID,omitempty"`
+	// HostIPC determines if the policy allows the use of HostIPC in the pod spec.
+	HostIPC bool `json:"hostIPC,omitempty"`
+	// SELinuxContext is the strategy that will dictate the allowable labels that may be set.
+	SELinuxContext SELinuxContextStrategyOptions `json:"seLinuxContext,omitempty"`
+	// RunAsUser is the strategy that will dictate the allowable RunAsUser values that may be set.
+	RunAsUser RunAsUserStrategyOptions `json:"runAsUser,omitempty"`
+
+	// The users who have permissions to use this policy
```

Comment:
Yes, this is correct.  Access is done via [a use verb](https://github.com/kubernetes/kubernetes/pull/33080#issuecomment-254277688) supported by rbac